### DISCUSSION TITLE
Opentofu or Terrafom

### DISCUSSION THREAD

**Discussion by fforootd** [zitadel/zitadel#7238] [Polls]
As you might be aware we have a [TF provider for ZITADEL](https://github.com/zitadel/terraform-provider-zitadel).

Recently we had more and more questions around our [Opentofu](https://opentofu.org/) "support".

Our question here is, what does the community interested more in the future. That we focus Opentofu or Terraform?

**Comment by fforootd** [comment]
So far opentofu clearly makes the race 😁

**Comment by andi0b** [comment]
For now I would say both? I think there is no clear "winner" yet.

**Reply by fforootd** [reply]
yeah, not many votes on this subject 😁 

looks like we need to try keeping it workable with both

**Reply by andi0b** [reply]
I don’t mean „winner“ of this poll. More like winner of the general competition of market share.


For example terraform cloud doesn’t support opentofu yet ;)

**Reply by fforootd** [reply]
Ahhhh, sorry for the misunderstanding, now I got you.

**Comment by danielloader** [comment]
Curveball request: I'd love a Crossplane provider based off the terraform provider - https://github.com/crossplane/upjet.

Just makes it simpler to provision helm charts of installations if I can throw Crossplane resources in the mix to set up the post fresh install zitadel - all the while keeping it all configured from Git.

**Comment by maulik13** [comment]
Hi, not sure how this goes in your roadmap but a crossplane provider would help a great deal for our team. Creating and maintaining a provider should not be a big issue when using the tooling to convert a terraform provider.

**Comment by kasir-barati** [comment]
@fforootd any update on this?
